update.sh

```diff
@@ -46,7 +46,8 @@ pgrep apt &>/dev/null && { echo "apt is currently running. Try again later";  ex
 # install new dependencies
 type jq &>/dev/null || {
   apt-get update
-  apt-get install -y --no-install-recommends jq
+  apt-get install -y --no-install-recommends jq tmux locales-all
+  mkdir -p "$(dirname $CONFDIR)/ncp-tmux"
```

Comment:
```suggestion
  mkdir -p "$(dirname "$CONFDIR")/ncp-tmux"
```

Reply:
Ah, that's actually not up to date anymore. I missed that (that's what I told you about - ncp.sh is on a later state than update.sh)

Reply:
Fixed in a357f2cdd45b97eb8685440b935ca5c00e30d4cc